superset/connectors/sqla/models.py

```diff
@@ -542,7 +542,7 @@ def get_sqla_query(  # sqla
             where = extras.get('where')
             if where:
                 where = template_processor.process_template(where)
-                where_clause_and += [sa.text('({})'.format(where))]
+                where_clause_and += [sa.text(u'({})'.format(where.encode('utf-8').decode('utf-8')))]
```

Comment:
I don't think this is python 3 compatible. Look into the `six` lib to write py2/3 compatible code